docs/MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md

```diff
@@ -68,6 +68,43 @@ Detailed information on how to convert models from the <a href="https://github.c
 |Faster R-CNN Inception ResNet V2 Low Proposals Open Images\*|  [faster_rcnn_inception_resnet_v2_atrous_lowproposals_oid_2018_01_28.tar.gz](http://download.tensorflow.org/models/object_detection/faster_rcnn_inception_resnet_v2_atrous_lowproposals_oid_2018_01_28.tar.gz)|
 |Faster R-CNN ResNet 101 AVA v2.1\*|  [faster_rcnn_resnet101_ava_v2.1_2018_04_30.tar.gz](http://download.tensorflow.org/models/object_detection/faster_rcnn_resnet101_ava_v2.1_2018_04_30.tar.gz)|
 
+**Supported Pre-Trained Topologies from TensorFlow 2 Object Detection Models Zoo**
+
+Detailed information on how to convert models from the <a href="https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_detection_zoo.md">TensorFlow 2 Detection Model Zoo</a> is available in the [Converting TensorFlow Object Detection API Models](tf_specific/Convert_Object_Detection_API_Models.md) chapter. The table below contains models from the Object Detection Models zoo that are supported.
```

Comment:
Does it refer to the same chapter as for TensorFlow 1 OD models? Are any relevant specifics of TensorFlow 2 OD models there that must be documented?

Reply:
Nothing special. Only different configuration files.